apps/emqx_dashboard/src/emqx_dashboard_collection.erl

```diff
@@ -52,6 +52,11 @@ start_link() ->
 
 get_collect() -> gen_server:call(whereis(?MODULE), get_collect).
 
+-spec select_data() -> [#mqtt_collect{}].
+select_data() ->
+    Time = emqx_dashboard_collection:get_universal_epoch() - 7200000,
```

Comment:
Where the 7200000 comes from?

Reply:
This constant comes from the old code, I just moved this function here